WordPress/build.gradle

```diff
@@ -54,9 +54,9 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "alpha-252"
+            versionName "alpha-253"
         }
-        versionCode 942
+        versionCode 945
```

Comment:
We should merge https://github.com/wordpress-mobile/WordPress-Android/pull/13259 first (`versionCode 943` **before** merging this PR, right?

Reply:
I would close #13259 which brings nothing new, other than a merge conflicts, and just merge this one.

Reply:
That sounds good to me. The hotfix is in 16.0, so the fix will make it into `develop`, which is what matters.

Closing the other PR now.